### ISSUE TITLE
numpy 2.0 compatibility

### ISSUE THREAD

**Issue by WillSH97** [changyaochen/rbo#23]
as per the title - deploying a pipeline/app with RBO in logging currently requires downgrading numpy version in requirements.txt

**Comment by CtrlShanya** [comment]
I agree, it would be great to get a version compatible with numpy 2.0, to use the current latest version, i have to downgrade the numpy for the whole package :/
